fastlane/lib/fastlane/actions/push_git_tags.rb

```diff
@@ -23,6 +27,15 @@ def self.description
 
       def self.available_options
         [
+          FastlaneCore::ConfigItem.new(key: :force,
+                                       env_name: "FL_PUSH_GIT_FORCE",
+                                       description: "Force push to remote. Defaults to false",
+                                       is_string: false,
+                                       default_value: false),
+          FastlaneCore::ConfigItem.new(key: :remote,
+                                       env_name: "FL_GIT_PUSH_REMOTE",
+                                       description: "The remote to push to. Defaults to `origin`",
+                                       default_value: 'origin')
```

Comment:
It hasn't been `origin` by default, I don't know how git handles this, however I'd prefer to not include this string in the `command` array at all if the user didn't provide is, so that we don't change the behavior 👍